### PR TITLE
Added `experimentalExtraAllowedOrigins` to config

### PR DESCRIPTION
**Pull request by awanlin**
## Hey, I just made a Pull Request!

Added `experimentalExtraAllowedOrigins` to config

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/auth-backend/config.d.ts [hunk 1]
```diff
@@ -191,5 +191,9 @@ export interface Config {
        */
       backstageTokenExpiration?: HumanDuration;
     };
+    /**
+     * Additional app origins to allow for authenticating
+     */
+    experimentalExtraAllowedOrigins?: string;
```
Comment (sennyeya) [inline plugins/auth-backend/config.d.ts:197]:
```suggestion
    experimentalExtraAllowedOrigins?: string[];
```
😁

**Review by sennyeya — COMMENTED** [review]
Nice! I wonder if there's something we could script here to make sure these show up in the `config.d.ts`. Maybe even type generation 😁 . Kind of related to #21430.

File: plugins/auth-backend/config.d.ts (hunk 1 above)
Comment (awanlin) [reply plugins/auth-backend/config.d.ts]: Oh, good catch, thanks and fixed!

**Comment by awanlin** [comment]
> Nice! I wonder if there's something we could script here to make sure these show up in the `config.d.ts`. Maybe even type generation 😁 . Kind of related to #21430.

Yes, that would be nice! No idea how we'd do it though, lol

**Review by Rugvip — APPROVED** [review]
:shipit:

**Comment by awanlin** [comment]
Thanks for the review @sennyeya and thanks for the merge @Rugvip 🚀

**Pull request merged**